bigtop-packages/src/rpm/hadoop/SPECS/hadoop.spec

```diff
@@ -623,7 +623,7 @@ fi
 #%config(noreplace) %{etc_hadoop}/conf.empty/mapred-site.xml
 %config(noreplace) %{etc_hadoop}/conf.empty/mapred-env.sh
 %config(noreplace) %{etc_hadoop}/conf.empty/mapred-queues.xml.template
-%config(noreplace) %{etc_hadoop}/conf.empty/mapred-site.xml.template
+%config(noreplace) %{etc_hadoop}/conf.empty/mapred-site.xml
```

Comment:
Not sure why remove the template here. Is it because this is also renamed by upstream?

Reply:
That is upstream change. It was done in https://github.com/apache/hadoop/commit/f1de2c8560a4ce1a346727c2c265570c5b6b872e

Reply:
Commit and pull request description has been changed to clarify question above.